packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -538,7 +538,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 runtimeVersion: pkgVersion,
             },
         });
-
+        const packingLevel = runtimeOptions?.summaryOptions?.disableIsolatedChannels ? 1 : 2;
```

Comment:
You should move the `combinedRuntimeOptions` generated below to here and use it instead. It doesn't matter today but if we add more default options, this may become incorrect.

Reply:
Also, add a comment why disableIsolatedChannels changes the value of packing level passed to BlobAggregateStorage.